IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -109,7 +109,16 @@ - (nullable instancetype)initWithGroup:(nullable NSString *)keychainGroup
     }
     
     MSIDKeychainUtil *keychainUtil = [MSIDKeychainUtil sharedInstance];
-    if (!keychainUtil.teamId) return nil;
+    if (!keychainUtil.teamId)
+    {
+        if (error)
+        {
+            *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"Failed to retrieve teamId from keychain.", nil, nil, nil, nil, nil);
+            MSID_LOG_WITH_CTX(MSIDLogLevelError, nil, @"Failed to retrieve teamId from keychain.");
```

Comment:
nit: please produce log regardless if error is passed or not

Reply:
Addressed.